tmt/cli.py

```diff
@@ -2053,17 +2062,18 @@ def clean_guests(
     if last and bool(id_):
         raise tmt.utils.GeneralError(
             "Options --last and --id cannot be used together.")
-    if not Path(workdir_root).exists():
+    workdir_root = Path(_workdir_root) if _workdir_root is not None else None
```

Comment:
I'm not sure if I understand it correctly, but do we need the temporary `_workdir_root`?   
I mean, shouldn't just `if workdir_root and not Path(workdir_root).exists():` be enough?

Reply:
It's on purpose. The idea is, the input is a string, but we want to convert it to `Path` as soon as possible, and use `Path` from that moment on. And `workdir_root` is a natural name for labeling this `Path`. Therefore we ask Click to store the string into `_workdir_root`, then we instantiate the `Path` instance, and assign it to `workdir_root`. If we had `workdir_root` only, a string, then we would need to think about converting it to `Path` when passing it on, e.g. to `tmt.Clean` below. This way, we convert the value to the proper internal type on the boundary.

Reply:
Thanks for your reply ,mentor:)